v2/pkg/genruntime/configmaps/validation.go

```diff
@@ -21,6 +22,11 @@ type keyPair struct {
 // ValidateDestinations checks that no two destinations are writing to the same configmap/key, as that could cause
 // those values to overwrite one another.
 func ValidateDestinations(destinations []*genruntime.ConfigMapDestination) (admission.Warnings, error) {
+	return ValidateDestinationsExt(destinations, nil)
+}
+
+// TODO: ValidateDestinationsExt will replace ValidateDestinations in a future PR.
```

Comment:
Hasn't this replacement effectively already happened? given the code already on Line 25? Why not do the replacement now?

Reply:
No not yet, what I mean by this is that there is a future PR coming where a generator change also happens. When that happens, `ValidateDestinations` (1 arg) goes away and becomes `ValidateDestinations` (2 args).

The advantage of doing it like this was just that it let me split the PRs, because the generator PR is going to be pretty big.